azkaban-web-server/src/main/java/azkaban/webapp/servlet/ExecutorServlet.java

```diff
@@ -1099,7 +1108,11 @@ private void ajaxRampActions(final HttpServletRequest req,
   UPDATE_ALLOW_LIST("updateAllowList"),
   APPEND_ALLOW_LIST("appendAllowList"),
   REMOVE_FROM_ALLOW_LIST("removeFromAllowList"),
-  UPDATE_RAMP_UP("updateRampUp");
+  UPDATE_RAMP_UP("updateRampUp"),
+  UPDATE_DENY_LIST("updateDenyList"),
```

Comment:
Do we need update?

Reply:
removed update

Reply:
I think removing updateRampUp is not in the scope of this PR. We can open a new pr to deal with this later.